test/performance/shmem_perf_suite/common.h

```diff
@@ -123,41 +264,40 @@ static void aligned_buffer_free(char * ptr_aligned)
 #endif
 }
 
-static inline int is_divisible_by_4(int num)
-{
+static inline 
+int is_divisible_by_4(int num) {
     assert(num >= 0);
     assert(sizeof(int) == 4);
     return (!(num & 0x00000003));
```

Comment:
This is not portable, because C doesn't require a particular representation for signed integers.  Unless this is in an innermost loop (looks like it's only used in diagnostic code), you can safely write `num % 4 == 0` and rely on the compiler to do the right thing.  Also, the assert non-negative should probably be `if (num < 0) return 0;` since assert should not be used for a persistent error check.

Reply:
Ok. Will update this.

Reply:
Actually, `shmem_global_exit` may be preferred instead of `return 0`.